Triphouse-project/index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
```

Comment:
All images should be in Triphouse-project folder.

Reply:
moved the pictures to the Triphouse-project folder